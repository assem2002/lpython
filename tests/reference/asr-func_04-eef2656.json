{
    "basename": "asr-func_04-eef2656",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/func_04.py",
    "infile_hash": "f8a6eed44ebd1dee435e6db842a874c41b8ba2b13d88d16944a4d265",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-func_04-eef2656.stderr",
    "stderr_hash": "e825889cde0d7fc3a1063aa39cb83348ac45de675aac96bdee345922",
    "returncode": 2
}