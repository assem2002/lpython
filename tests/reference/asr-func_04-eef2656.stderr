semantic error: Modifying input function parameter `l` is not allowed
 --> tests/errors/func_04.py:4:5
  |
4 |     l.append(5)
  |     ^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
